src/tests/ftest/io/nvme_object.yaml

```diff
@@ -0,0 +1,45 @@
+# change host names to your reserved nodes, the
+# required quantity is indicated by the placeholders
+
+hosts:
+ test_servers:
+  - boro-A
+  - boro-B
+  - boro-C
+  - boro-D
+  - boro-E
+  - boro-F
+  - boro-G
+ test_clients:
+  - boro-H
+timeout: 16000
```

Comment:
This is a **really** long timeout for a test that is supposed to run on PRs!  It is 4.4 hours!  That means that every time this test fails and has to timeout, we lose over 4 hours of cluster time.  This is exactly the sort of thing that leads to long queue backlogs.  Does this timeout reflect the actual amount of time this test is expected to run?  If it does, I don't think we can afford to run 4 hour tests on PRs.

Looking at the [test result](https://build.hpdd.intel.com/job/daos-stack/job/daos/job/master/1289/testReport/(root)/NvmeObject/) where this landed, it seems the test only took 13 minutes.  So why a 16000 second timeout?

I have had to kill a number of instances of this test over this past weekend just so that we can get the weekly test runs complete and not start the week out with a long queue.

Please ensure that test timeouts reflect how long a test should really take to run and not just use arbitrarily **HUGE** values because when those tests do fail, as this one has been doing all weekend, it ties up clusters for much too long and builds huge queue backlogs.